api/db/db_mgmt.js

```diff
@@ -129,6 +129,7 @@ let db_mgmt_module = function () {
                 registration_date: util.mysql_iso_time(new Date(Date.now())),
                 grad_date: new_account.grad_date,
                 mass_mail_optin: new_account.in_mailing_list,
+                total_meetings: 0
```

Comment:
0?

Reply:
this is when the account get initialized, new accounts start with 0 points